plugins/sbt/graphql-java-codegen-sbt-plugin/src/main/scala/io/github/dreamylost/graphql/codegen/GraphQLCodegenKeys.scala

```diff
@@ -0,0 +1,98 @@
+package io.github.dreamylost.graphql.codegen
+
+import java.util
+
+import com.kobylynskyi.graphql.codegen.model.{ ApiNamePrefixStrategy, ApiRootInterfaceStrategy }
+import sbt._
+
+/**
+ *
+ * @author liguobin@growingio.com
+ * @version 1.0,2020/7/15
+ */
+trait GraphQLCodegenKeys {
+
+  //Conflict with SBT key
+  val genPackageName = settingKey[Option[String]]("packageName")
```

Comment:
Please align the name of this config with all other configs by renaming it to `generatePackageName`.